src/styles/_catalog.scss

```
@@ -14,9 +14,17 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
-  }
+    transition-property: transform, color;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
+    
+    &:hover {
+      transform: scale(120%);
```

```suggestion
      transform: scale(1.2);
```

Наступного разу використовуй відносні значення